contracts/hyphen/LiquidityPool.sol

```diff
@@ -193,6 +193,29 @@ contract LiquidityPool is
             incentivePool[tokenAddress];
     }
 
+    /**
+     * @dev Function used to set the correct value of CurrentLiqudity variable defined in LiquidityProvider.sol
+     * by calculating reward, accumulatedFee, total Fee & available token balance in the pool
+     * @param tokenAddress of which corect value needs to be set
+     */
+    function setCurrentLiquidity(address tokenAddress) public onlyOwner {
+        uint256 tokenLiquidity = 0;
+        if (tokenAddress == NATIVE) {
+            tokenLiquidity = address(this).balance;
+
+        } else {
+            IERC20Upgradeable baseToken = IERC20Upgradeable(tokenAddress);
+            tokenLiquidity =  baseToken.balanceOf(address(this));
+        }
+
+        uint256 availableLiquidity = tokenLiquidity -
```

Comment:
we assume that AL includes all the fee in other function (check getCurrentLiquidity) therefore I don't think we need to subtract stored fee here